packages/ember-htmlbars/tests/helpers/custom_helper_test.js

```diff
@@ -23,36 +23,6 @@ QUnit.module('ember-htmlbars: custom app helpers', {
   }
 });
 
-QUnit.test('dashed shorthand helper is resolved from container', function() {
```

Comment:
At first I thought the "dashed" in the name implies that we have another "test matrix" situation here, but turns out literally all the tests in this file is prefixed with "dashed"